### DISCUSSION TITLE
Prefix all filament component config files with "filament-"

### DISCUSSION THREAD

**Discussion by robertgdev** [filamentphp/filament#4447] [Ideas]
For v3, it would be nice if the config files for tables, forms and notifications would be installed (when published) under the config directory with the 'filament-' prefix ... so the resulting filenames under the config directory would be

- filament-forms
- filament-notifications
- filament-tables

This would clarify the fact that these files are part of the filament suite.

**Comment by danharrin** [comment]
We will consider this, thanks
